api/providers/views.py

```diff
@@ -595,3 +596,25 @@ def perform_destroy(self, instance):
             self.get_provider().remove_from_group(instance, instance.permission_group)
         except ValueError as e:
             raise ValidationError(str(e))
+
+
+class RegistrationProviderSchemaList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin):
+    permission_classes = (
+        base_permissions.TokenHasScope,
```

Comment:
Probably should also have a `drf_permissions.IsAuthenticatedOrReadOnly,` class here. I'm surprised this isn't causing a test to fail.

Reply:
yeah it's a issue with testmon where changes in the tests themselves aren't always picked up, it failed for me on my local branch.